Add Granular Override Warning

FKLC opened the pull request:
The user can dismiss the the notification, but they won't be able to modify the target using the extension. The user can manage their granular overrides, we don't want to overwrite them. This patch also updates deps, so it fixes #9 too.

FKLC committed:
Show warning if the user has a granular override for the website

FKLC committed:
Change min-width to width so that notifications don't change the window size

FKLC committed:
Update deps

tomrittervg commented:
I would change the text to `Granular overrides affecting this domain were found. The extension does not support them, and behavior may be unexpected.`

FKLC committed:
Update warning message and deps

FKLC committed:
Revert "Update deps"

This reverts commit b95676e7f9a73f69a24953741cb09221e495cca8.

FKLC committed:
Only update required deps

FKLC commented:
Updating the deps caused node version mismatch. My local environment is running node 22 while the build-fingerprinting-protection-debugger image runs node version 16. So I reverted changes and updated only required deps.

FKLC committed:
update task image and deps

FKLC commented:
Found an even better solution. Changing the task image
